### PR TITLE
Decimal handling

### PR DESCRIPTION
**Pull request by Avene**

### CODE REVIEW

File: simple_salesforce/tests/test_api.py [hunk 1]
```diff
@@ -604,7 +604,111 @@ def test_query(self):
...
+                 '"done": false, "nextRecordsUrl": '
+                 '"https://example.com/query/next-records-id", "totalSize": 1}',
+            status=http.OK)
+        session = requests.Session()
+        client = Salesforce(session_id=tests.SESSION_ID,
+                            instance_url=tests.SERVER_URL,
+                            session=session)
+
+        result = client.query('SELECT ID, Price FROM Account')
+        print(result)
```
Comment (digitalaun) [inline simple_salesforce/tests/test_api.py]: stray print statement

File: simple_salesforce/tests/test_api.py [hunk 2]
```diff
@@ -604,7 +604,111 @@ def test_query(self):
...
+                    OrderedDict([
+                        ('ID', '5'), ('Price', -1.1234567890123456),
+                    ]),
+                    OrderedDict([
+                        ('ID', '6'), ('Price', -12345678901234567),
+                    ]),
+                ]),
+                ('done', False),
+                ('nextRecordsUrl', "https://example.com/query/next-records-id"),
+                ('totalSize', 1),
```
Comment (digitalaun) [inline simple_salesforce/tests/test_api.py]: While not used by the test, `totalSize` should match the actual number of records (six)

File: simple_salesforce/tests/test_api.py [hunk 3]
```diff
@@ -604,7 +604,111 @@ def test_query(self):
...
+                        ('Price', decimal.Decimal('-1.1234567890123456')),
+                    ]),
+                    OrderedDict([
+                        ('ID', '6'),
+                        ('Price', decimal.Decimal('-12345678901234567')),
+                    ]),
+                ]),
+                ('done', False),
+                ("nextRecordsUrl", "https://example.com/query/next-records-id"),
+                ('totalSize', 1),
```
Comment (digitalaun) [inline simple_salesforce/tests/test_api.py]: Total size should be 6

**Review by digitalaun — CHANGES_REQUESTED** [review]
Thanks for continuing this. A couple of minor changes needed.

Did you get tests passing locally? I had a couple of unrelated tests failing due to them bypassing normal initialization.

**Comment by digitalaun** [comment]
I'm getting test failures when running them locally using `tox`
```
  File "/Users/scott/MakeLeaps/simple-salesforce/simple_salesforce/tests/test_api.py", line 652, in test_query_parse_float
    ('totalSize', 6),
AssertionError: Order[384 chars]https://example.com/query/next-records-id'), ('totalSize', 1)]) != Order[384 chars]https://example.com/query/next-records-id'), ('totalSize', 6)])
```

**Comment by Avene** [comment]
🤦 I just forgot update the test condition. will fix later. 🤕

**Comment by Avene** [comment]
Closing this PR on my repo, and let's see ML repo instead https://github.com/makeleaps/simple-salesforce/pull/2

**Pull request closed without merging**